pep-0544.txt

```diff
@@ -949,6 +1048,44 @@ this in type checkers for non-protocol classes could be difficult. Finally, it
 will be very easy to add this later if needed.
 
 
+Prohibit explicit subclassing of protocols by non-protocols
+-----------------------------------------------------------
+
+This was rejected mainly for two reasons:
+
+* Backward compatibility: People are already using ABCs, including generic
+  ABCs from ``typing`` module. If we prohibit explicit subclassing of these
+  ABCs, then quite a lot of code will break.
+
+* Convenience: There are existing protocol-like ABCs (that will be turned
+  into protocols) that have many useful "mix-in" (non-abstract) methods.
+  For example in the case of ``Sequence`` one only needs to implement
+  ``__getitem__`` and ``__len__`` in an explicit subclass, and one gets
+  ``__iter__``, ``__contains__``, ``__reversed__``, ``index``, and ``count``
+  for free.
+
+
+Backwards Compatibility
+=======================
+
+This PEP is fully backwards compatible.
```

Comment:
If we turn `Sequence` and friends into runtime protocols, the results of some `isinstance` checks are going to change in some edge cases. (Also, the performance of these `isinstance` checks might be affected, though not sure about it.)